apps/dashboard/src/components/primitives/form/faceted-filter/components/base-filter-content.tsx

```diff
@@ -0,0 +1,74 @@
+import { SizeType } from '../types';
+import { FilterInput } from './filter-input';
+import { ClearButton } from './clear-button';
+import { RiArrowDownLine, RiArrowUpLine } from 'react-icons/ri';
+import { EnterLineIcon } from '../../../../icons/enter-line';
+
+interface BaseFilterContentProps {
+  inputRef: React.RefObject<HTMLInputElement>;
+  title?: string;
+  onClear: () => void;
+  size: SizeType;
+  hideSearch?: boolean;
+  hideClear?: boolean;
+  searchValue?: string;
+  onSearchChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  searchPlaceholder?: string;
+  showNavigationFooter?: boolean;
+  showEnterIcon?: boolean;
+  children?: React.ReactNode;
+}
+
+export function BaseFilterContent({
+  inputRef,
+  title,
+  onClear,
+  size,
```

Comment:
This folder is a new Form MultiFacted component that supports multiple field types

![CleanShot 2024-12-10 at 13 55 45@2x](https://github.com/user-attachments/assets/eb40bfbe-56c2-47c7-902a-258f35a6a629)